### ISSUE TITLE
To complete the TupleDesc.java class

### ISSUE THREAD

**Issue by RosaGeorge31** [ishaan-6298/SimpleDB#2]
The second class we need to fix is the TupleDesc class. After this one is complete we need to ensure that code passes the unit tests TupleTest and TupleDescTest.

**Comment by salman-bhai** [comment]
Assign someone the Issues as well!

**Comment by BhavishyaV** [comment]
I have completed most of the skeleton methods in Tupledesc()
The method public boolean equals(Object o) is incomplete
Also, I'm not sure what exactly  (public Iterator<TDItem> iterator() )has to do. I have coded for it though.
Another thing is that I'm currently getting 19 errors of the same kind and can' seem to fix them.
If some one could look into it with a fresh perspective, it would be great.
I have attached the file and a screenshot of the errors I'm getting.
[TupleDesc.txt](https://github.com/ishaan-6298/SimpleDB/files/1066207/TupleDesc.txt)
![errortd](https://user-images.githubusercontent.com/26907203/27007813-9f733594-4e7d-11e7-9377-9452fd7add09.png)
